adage/analyze/management/commands/import_gene_sample_expr.py

```diff
@@ -40,12 +58,21 @@ def handle(self, **options):
                 "values: %s" % e)
 
 
-def import_expr(file_handle):
+def import_expr(file_handle, organism_tax_id):
     """
     Function that reads input file and load gene-sample expression values
     into the database.
     """
 
+    # Make sure input organism_tax_id already exists in database.
+    try:
+        organism = Organism.objects.get(taxonomy_id=organism_tax_id)
+    except Organism.DoesNotExist:
+        raise Exception("Input organism_tax_id is not found in the database. "
+                        "Please use the management command "
+                        "'organism_create_or_update.py' in djago-organism "
+                        "package to create this organism.")
```

Comment:
Just noticed the typo: `djago-organism` should be `django-organisms`. I will fix it.